### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by linsun**
Update the warning msg, see PR for details

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 </h1>
 
 
-# 🚧 NOTE: This project is in the process of being donated to the CNCF. It is being re-factored as part of the process. 🚧
+# 🚧 NOTE: This project is in the process of being donated to the CNCF and not affiliated with the Kubernetes project. 🚧
```
Comment (linsun) [inline README.md:12]:
```suggestion
# 🚧 NOTE: This project is in the process of being donated to the CNCF and is not affiliated with the Kubernetes project. 🚧
```

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@
 </h1>
 
 
-# 🚧 NOTE: This project is in the process of being donated to the CNCF. It is being re-factored as part of the process. 🚧
+# 🚧 NOTE: This project is in the process of being donated to the CNCF and is not affiliated with the Kubernetes project. 🚧
```
Comment (ilrudie) [inline README.md:12]: IMO, it makes sense to leave the note about refactoring. It's still valid.
```suggestion
# 🚧 NOTE: This project is in the process of being donated to the CNCF and is not affiliated with the Kubernetes project. It is being re-factored as part of the process.🚧
```

**Review by ilrudie — APPROVED** [review]
Seems fine but I do have a suggestion.

File: README.md (hunk 2 above)
Comment (yuval-k) [reply README.md:12]: yea i want to include the re-factored part, to make sure people are aware

Comment (yuval-k) [reply README.md:12]: otherwise, lgtm

**Pull request merged**